requirements.txt

```diff
@@ -1,4 +1,5 @@
 click
+jsons
```

Comment:
I think we should keep to the standard library for JSON support.  cc @kalxas for comments.

Reply:
+1
we should avoid adding new dependencies if possible